POS/Overview/01_General.md

```diff
@@ -1,4 +1,5 @@
 # General information
+<!---Error-->
 
```

Comment:
Wir sollten Overview (Benefits and Features) mit den neuen Funktionen aktualisieren, solang sie standard sind (ich meine, nicht nur für FC K)
Terminologie: inventory/inventories ist es nicht mehr erlaubt -> stock?